src/get_family_names.py

```diff
@@ -139,7 +139,7 @@ def write_raw_data(df):
         'id': 'congressperson_id',
         'Filiação': 'parents'})
 
-    if len(people_with_more_than_two_parents):
+    if people_with_more_than_two_parents:
```

Comment:
As this `people_with_more_than_two_parents` is a Pandas data frame, this change breaks the code:

```
Traceback (most recent call last):
  File "src/get_family_names.py", line 183, in <module>
    get_congresspeople_parents_names()
  File "src/get_family_names.py", line 179, in get_congresspeople_parents_names
    write_raw_data(df)
  File "src/get_family_names.py", line 142, in write_raw_data
    if people_with_more_than_two_parents:
  File "/usr/local/var/pyenv/versions/anaconda3-4.1.1/envs/serenata_de_amor/lib/python3.5/site-packages/pandas/core/generic.py", line 917, in __nonzero__
    .format(self.__class__.__name__))
ValueError: The truth value of a DataFrame is ambiguous. Use a.empty, a.bool(), a.item(), a.any() or a.all().
```

Would you mind rolling back or editing your code to work with Pandas data frame?